### PR TITLE
Upgrade to Open Enclave v0.8.2

### PR DESCRIPTION
**Pull request by podcastinator**
Open Enclave v0.8.2 adds a couple of security improvements, incorporated in this PR:
1. Stack protection
2. LVI vulnerability mitigation

### CODE REVIEW

File: tests/travis/run_test.sh
Comment (chester-leung) [inline tests/travis/run_test.sh:50]: Any reason we disable LVI mitigation for all build tests?

Comment (podcastinator) [reply tests/travis/run_test.sh:50]: Currently, it requires some messing around with the OE SDK installation -- it's not enough to simply install the latest SDK; there are a whole bunch of other steps required for installing and using the LVI mitigated versions of the OE libraries.
I'm hoping that the OE maintainers will simplify the build process in a future version of the SDK. Perhaps we can hold off on this until then?

Comment (chester-leung) [reply tests/travis/run_test.sh:50]: Makes sense!

**Review by chester-leung — APPROVED** [review]

**Comment by chester-leung** [comment]
Thanks!

**Pull request merged**
